core/trino-main/src/main/java/io/trino/sql/planner/plan/Assignments.java

```diff
@@ -60,6 +60,11 @@ public static Assignments identity(Iterable<Symbol> symbols)
         return builder().putIdentities(symbols).build();
     }
 
+    public static boolean isIdentity(Symbol symbol, Expression expression)
```

Comment:
Please move this method to `InlineProjections`, also I would rename the method to `isSymbolReference`

Reply:
Moved and renamed.